src/app/features/tech-record/components/tech-record-summary/tech-record-summary.component.ts

```diff
@@ -76,7 +76,11 @@ export class TechRecordSummaryComponent implements OnInit {
         this.psvBrakeTemplate = PsvBrakeSection;
         this.brakeTemplateWheelsNotLocked = PsvBrakeSectionWheelsNotLocked;
         this.brakeTemplateWheelsHalfLocked = PsvBrakeSectionWheelsHalfLocked;
-        this.dimensionsTemplate = getDimensionsSection(VehicleTypes.PSV, this.vehicleTechRecord.noOfAxles, this.vehicleTechRecord?.dimensions?.axleSpacing);
+        this.dimensionsTemplate = getDimensionsSection(
+          VehicleTypes.PSV,
+          this.vehicleTechRecord.noOfAxles,
+          this.vehicleTechRecord?.dimensions?.axleSpacing
```

Comment:
(I realise this is not your change) Is there a chance vehicleTechRecord could be null? The optional chaining operator is used inconsistently on these two lines and further below.

Reply:
A solid point, it can be null as far as I know, I will go an add these in to all of them.